### PR TITLE
Add registrar to sandboxes.

### PR DESCRIPTION
**Pull request by iloveagent57**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

File: playbooks/roles/registrar/defaults/main.yml [hunk 1]
```diff
@@ -58,6 +60,8 @@ REGISTRAR_URL_ROOT: 'http://{{ REGISTRAR_DOMAIN }}:{{ REGISTRAR_NGINX_PORT }}'
 REGISTRAR_API_ROOT: '{{ REGISTRAR_URL_ROOT }}/api'
 REGISTRAR_LOGOUT_URL: '{{ REGISTRAR_URL_ROOT }}/logout/'
 
+REGISTRAR_SECRET_KEY: 'YOUR SECRET KEY HERE'
```
Comment (iloveagent57) [inline playbooks/roles/registrar/defaults/main.yml]: Remove this from here, add it in the sandbox-internal repo.

File: playbooks/edx_continuous_integration.yml [hunk 2]
```diff
@@ -12,7 +12,6 @@
       - cms
       - lms
       - xqueue
-      - xserver
```
Comment (iloveagent57) [inline playbooks/edx_continuous_integration.yml:15]: Also - remember to remove the xserver stuff from sandbox internal

File: util/jenkins/ansible-provision.sh [hunk 3]
```diff
@@ -181,6 +181,15 @@ if [[ -z $journals_version ]]; then
   journals_version="master"
 fi
 
+# if [[ -z $registrar ]]; then
+#   registrar="false"
+# fi
+registrar="true"
```
Comment (fredsmith) [inline util/jenkins/ansible-provision.sh]: by commenting out the test above, and just hardcoding registrar=true, you can never not provision registrar.

**Review by fredsmith — CHANGES_REQUESTED** [review]
I'm not sure if you meant to put that test back in.  Otherwise, looks good.

**Review by fredsmith — APPROVED** [review]
LGTM :shipit:

**Review by fredsmith — APPROVED** [review]

**Pull request merged**